{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--ff7b6f65-dad6-4faf-aea2-d6c85b227fa0",
    "created": "2024-08-13T16:13:59.586579Z",
    "modified": "2024-08-13T16:13:59.586583Z",
    "name": "No Title Available",
    "description": "Pillow before 3.3.2 allows context-dependent attackers to obtain sensitive information by using the \"crafted image file\" approach, related to an \"Integer Overflow\" issue affecting the Image.core.map_buffer in map.c component.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2016-9189"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://github.com/python-pillow/Pillow/pull/2146/commits/c50ebe6459a131a1ea8ca531f10da616d3ceaa0f"
        },
        {
            "name": "GLSA-201612-52",
            "tags": [
                "vendor-advisory",
                "x_refsource_GENTOO"
            ],
            "url": "https://security.gentoo.org/glsa/201612-52"
        },
        {
            "name": "94234",
            "tags": [
                "vdb-entry",
                "x_refsource_BID"
            ],
            "url": "http://www.securityfocus.com/bid/94234"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://github.com/python-pillow/Pillow/issues/2105"
        },
        {
            "name": "DSA-3710",
            "tags": [
                "vendor-advisory",
                "x_refsource_DEBIAN"
            ],
            "url": "http://www.debian.org/security/2016/dsa-3710"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "http://pillow.readthedocs.io/en/3.4.x/releasenotes/3.3.2.html"
        }
    ]
}